readthedocs/config/config.py

```diff
@@ -806,6 +815,30 @@ def validate_sphinx(self):
 
         return sphinx
 
+    def validate_final_doc_type(self):
+        """
+        Validates that the doctype is the same as the admin panel.
+
+        This a temporal validation, as the configuration file
+        should support per version doctype, but we need to
+        adapt the rtd code for that.
+        """
+        dashboard_doctype = self.defaults.get('doctype', 'sphinx')
+        if self.doctype != dashboard_doctype:
+            key = 'mkdocs' if self.doctype == 'mkdocs' else 'sphinx'
+            needed_doctype = (
+                'mkdocs' if dashboard_doctype == 'mkdocs' else 'sphinx'
+            )
+            self.error(
+                key,
+                'Your project is configured as "{doctype}" in your admin '
```

Comment:
`doctype` here is the interpolation that could be displayed as `sphinx_htmldir`. `key` in this string can only be `sphinx` or `mkdocs`.